lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,24 @@
 module CC
   module Engine
     class AnalyzableFiles
```

Comment:
Do we still need this class now that things are simpler? It seems like its only functions are to:

- raise an error if there are no include_paths
- return include_paths (only directories and .coffee files)